pinot-server/src/main/java/org/apache/pinot/server/api/resources/TablesResource.java

```diff
@@ -289,7 +289,7 @@ public String uploadLLCSegment(
             String.format("Failed to upload table %s segment %s to segment store", realtimeTableName, segmentName),
             Response.Status.INTERNAL_SERVER_ERROR);
       }
-      return segmentDownloadUrl.getPath();
+      return segmentDownloadUrl.toString();
```

Comment:
why this change? usually `toString` is for debugging only.

Reply:
This endpoint was introduced in the dependent pr: #6653 . 
URI.getPath() will return a truncated url, and the previous test case failed to catch it.
The test case is changed in this pr as well, in pinot-server/src/test/java/org/apache/pinot/server/api/BaseResourceTest.java